packages/api/src/auth/github/github.repository.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+
+import { RegisteredUser } from '@coderscamp/shared/models/user';
+
+import { GithubRepositoryPort } from '../../contracts/github.repository';
+
+@Injectable()
+export class GithubRepository {
+  constructor(private readonly repository: GithubRepositoryPort) {}
+
+  async findUserById(id: number) {
+    return this.repository.findById(id);
+  }
+
+  async findUserByGithubId(id: number) {
+    return this.repository.findByGithubId(id);
+  }
+
+  async createUser(data: Omit<RegisteredUser, 'id'>) {
```

Comment:
I recommend use of something like UUID as ID and get rid of handling situation when id is omitted. 
It's also better for app scalability than leaving generating id for the database.